### PR TITLE
[MTM-61288] add KPI widget documentation to widget collection

### PR DESCRIPTION
**Pull request by reey**

### CODE REVIEW

**Review by jdre-c8y — APPROVED** [review]

File: content/cockpit/widgets-collection.md [hunk 1]
```diff
@@ -265,11 +265,28 @@ If you want to use a link in the **HTML code** field, for example a link to a da
   <a style="cursor:pointer;" onclick="location.hash = '#/group/<<group-id>>/dashboard/<<dashboard-id>>'">link to another dashboard</a><br />
 ```
 
+### KPI {#kpi}
+
```
Comment (MWindel) [inline content/cockpit/widgets-collection.md:269]: @reey What is the "KPI" widget? Could you add 1-2 sentences to give the user an idea, what this widget is.

File: content/cockpit/widgets-collection.md [hunk 2]
```diff
@@ -265,11 +265,28 @@ If you want to use a link in the **HTML code** field, for example a link to a da
...
+The "KPI" widget visualizes a data point as a label. It is meant as a simple visualization for mainly KPIs (Key Performance Indicators).
```
Comment (MWindel) [inline content/cockpit/widgets-collection.md]:
```suggestion
The "KPI" (Key Performance Indicators) widget visualizes a data point as a label. This allows you to create a visualization for main KPIs.
```

File: content/cockpit/widgets-collection.md [hunk 3]
```diff
@@ -265,11 +265,28 @@ If you want to use a link in the **HTML code** field, for example a link to a da
...
+
+![KPI widget](/images/users-guide/cockpit/cockpit-widget-kpi.png)
+
+**Parameters to configure**
+
+On the left side, you can select the data point to be displayed.
```
Comment (MWindel) [inline content/cockpit/widgets-collection.md]:
```suggestion
On the left side, select the data point you want to display.
```

File: content/cockpit/widgets-collection.md [hunk 4]
```diff
@@ -265,11 +265,28 @@ If you want to use a link in the **HTML code** field, for example a link to a da
...
+You have to select exactly one active datapoint to create the "KPI" widget.
```
Comment (MWindel) [inline content/cockpit/widgets-collection.md]: @reey Why "exactly"? What happens when the user selects multiple data points?

File: content/cockpit/widgets-collection.md [hunk 5]
```diff
@@ -265,11 +265,28 @@ If you want to use a link in the **HTML code** field, for example a link to a da
...
+
+On the right side, you can adjust how the datapoint is going to be displayed. This includes:
```
Comment (MWindel) [inline content/cockpit/widgets-collection.md]:
```suggestion
On the right side, you can adjust how the data point is going to be displayed. This includes:
```

File: content/cockpit/widgets-collection.md [hunk 6]
```diff
@@ -265,11 +265,28 @@ If you want to use a link in the **HTML code** field, for example a link to a da
...
+* Icon: the icon to be displayed next to the data point
```
Comment (MWindel) [inline content/cockpit/widgets-collection.md]:
```suggestion
* Icon: The icon to be displayed next to the data point
```

File: content/cockpit/widgets-collection.md (hunk 2 above)
Comment (reey) [reply content/cockpit/widgets-collection.md]:
```suggestion
The "KPI" (Key Performance Indicators) widget visualizes a data point as a label. Such a data point could for example be the temperature of a device.
```

File: content/cockpit/widgets-collection.md (hunk 4 above)
Comment (reey) [reply content/cockpit/widgets-collection.md]: Then you can't save the configuration.

Comment (MWindel) [reply content/cockpit/widgets-collection.md]:
```suggestion
You must select only one active datapoint to create the "KPI" widget. If you select multiple data points at once, you cannot save the configuration
```

File: content/cockpit/widgets-collection.md [hunk 7]
```diff
@@ -265,11 +265,28 @@ If you want to use a link in the **HTML code** field, for example a link to a da
   <a style="cursor:pointer;" onclick="location.hash = '#/group/<<group-id>>/dashboard/<<dashboard-id>>'">link to another dashboard</a><br />
 ```
 
+### KPI {#kpi}
+
+The "KPI" (Key Performance Indicators) widget visualizes a data point as a label. Such a data point could for example be the temperature of a device.
```
Comment (MWindel) [inline content/cockpit/widgets-collection.md]:
```suggestion
The "KPI" (Key Performance Indicators) widget visualizes a data point as a label, for example, a data point for the temperature of a device.
```

File: content/cockpit/widgets-collection.md [hunk 8]
```diff
@@ -265,11 +265,28 @@ If you want to use a link in the **HTML code** field, for example a link to a da
...
+
+![KPI widget](/images/users-guide/cockpit/cockpit-widget-kpi.png)
+
+**Parameters to configure**
+
+On the left side, select the data point you want to display.
+You must select only one active datapoint to create the "KPI" widget. If you select multiple data points at once, you cannot save the configuration
```
Comment (MWindel) [inline content/cockpit/widgets-collection.md]:
```suggestion
You must select only one active datapoint to create the "KPI" widget. If you select multiple data points at once, you cannot save the configuration.
```

File: content/cockpit/widgets-collection.md [hunk 9]
```diff
@@ -265,11 +265,28 @@ If you want to use a link in the **HTML code** field, for example a link to a da
   <a style="cursor:pointer;" onclick="location.hash = '#/group/<<group-id>>/dashboard/<<dashboard-id>>'">link to another dashboard</a><br />
 ```
 
+### KPI {#kpi}
+
+The "KPI" (Key Performance Indicators) widget visualizes a data point as a label. Such a data point could for example be the temperature of a device.
+
+![KPI widget](/images/users-guide/cockpit/cockpit-widget-kpi.png)
+
+**Parameters to configure**
```
Comment (reey) [inline content/cockpit/widgets-collection.md:274]:
```
**Parameters to configure**
```

Is used in the whole file..

**Review by MWindel — APPROVED** [review]

**Pull request merged**